docs/examples/events/on_decorator03.py

```diff
@@ -0,0 +1,38 @@
+from textual import on
+from textual.app import App, ComposeResult
+from textual.widgets import Button
+
+
+class OnDecoratorApp(App):
+    CSS_PATH = "on_decorator.css"
+
+    def compose(self) -> ComposeResult:
+        """Three buttons."""
+        yield Button("Bell", id="bell")
+        yield Button("Toggle dark", classes="toggle dark")
+        yield Button("Quit", id="quit")
+
+    @on(Button.Pressed, button="#bell")  # (1)!
```

Comment:
I don't think we should repeat annotation. Just annotate anything changed / added.